NAMESPACE

```diff
@@ -109,9 +115,11 @@ S3method(st_interpolate_aw,stars)
 S3method(st_intersects,stars)
 S3method(st_join,stars)
 S3method(st_mosaic,character)
+S3method(st_mosaic,nc_proxy)
 S3method(st_mosaic,stars)
 S3method(st_mosaic,stars_proxy)
 S3method(st_normalize,stars)
+S3method(st_redimension,nc_proxy)
```

Comment:
For now, redimension and mosaic throw errors for nc_proxy as it will error. Would you rather put the error in the `stars_proxy` method or have a method that only ever errors?